### DISCUSSION TITLE
My name is AbdulMelik Mirac

### DISCUSSION THREAD

**Discussion by as123sa321** [community/community#60506] [GitHub Education]
### Hi everyone,

Hello, my name is abdulmelik mirac baydar I have a lot of merkım to the software and I came here and I am very happy.

**Comment by samus-aran** [comment]
👋 Welcome to the GitHub Community on Discussions, @as123sa321  Happy to have you here 🙂

To help keep our space clutter-free, we recommend introducing yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204) and checking out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776). This post will be removed within 24-48 hours to keep our community nice and tidy 🧹.
